nvflare/edge/device/android/sdk/core/FlareRunner.kt

```diff
@@ -0,0 +1,266 @@
+package com.nvidia.nvflare.sdk.core
+
+import android.util.Log
+import com.nvidia.nvflare.sdk.config.processTrainConfig
+import com.nvidia.nvflare.sdk.core.Context
+import com.nvidia.nvflare.sdk.core.Signal
+import com.nvidia.nvflare.sdk.core.ContextKey
+import com.nvidia.nvflare.sdk.core.DataSource
+import com.nvidia.nvflare.sdk.core.Filter
+import com.nvidia.nvflare.sdk.core.DXO
+import com.nvidia.nvflare.sdk.core.EventType
+import com.nvidia.nvflare.sdk.core.Executor
+
+/**
+ * Main orchestrator for federated learning on edge devices.
+ * Handles job fetching, task execution, and result reporting.
+ */
+abstract class FlareRunner(
```

Comment:
maybe we can just have one class AndroidFlareRunner instead of AndroidFlareRunner and FlareRunner?